src/components/NavigationBar.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import { Nav, Navbar, NavDropdown } from 'react-bootstrap';
+import { Link, NavLink } from 'react-router-dom';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faHome } from '@fortawesome/free-solid-svg-icons';
+import { faHtml5, faCss3, faJs } from '@fortawesome/free-brands-svg-icons';
+
+const NavigationBar = () => {
+  return (
+    <Navbar collapseOnSelect expand="lg" bg="primary" variant="dark">
+      <Navbar.Brand as={Link} to="/">
+        CYF Quiz
+      </Navbar.Brand>
+      <Navbar.Toggle aria-controls="responsive-navbar-nav" />
+      <Navbar.Collapse id="responsive-navbar-nav">
+        <Nav>
+          <Nav.Link as={NavLink} to="/" className="ml-5">
+            <FontAwesomeIcon
+              className="mr-2"
+              aria-hidden="true"
+              aria-label="Home icon"
+              icon={faHome}
+            />
+            Home
+          </Nav.Link>
```

Comment:
We can remove the `Home` link since most websites don't have one and will redirect to the home page when the user clicks on the logo

Reply:
Removed!